packages/cli/src/commands/build.ts

```diff
@@ -555,7 +555,150 @@ async function doBuild(
     builds: builderRoutes,
   });
 
-  const mergedImages = mergeImages(buildResults.values());
+  const images = vercelConfig?.images
+  if (images) {
```

Comment:
Should we do most of this with schema validation instead?

Can we extract this into a function like `validateImages`? It would make it easier to add tests for the validation error cases.

Reply:
Sounds good on both counts. I think this logic was lifted from `@vercel/next`.

Reply:
We have vercel.json schema validation for `vc dev` here.
So we should probably do the same for `vc build` (and add images schema 🙂 )

https://github.com/vercel/vercel/blob/e5131d55575096beb764e11ae2ff5e19ecaf940e/packages/cli/src/util/dev/validate.ts#L19-L34